### PR TITLE
chore: upgrade to rust 1.80

### PR DESCRIPTION
**Pull request by satyarohith**

### CODE REVIEW

**Review by dsherret — APPROVED** [review]
LGTM

File: cli/errors.rs [hunk 1]
```diff
@@ -112,17 +111,5 @@ pub fn get_error_class_name(e: &AnyError) -> &'static str {
...
-          "Error '{}' contains boxed error of unknown type:{}",
-          e,
-          e.chain().fold(String::new(), |mut output, e| {
-            let _ = write!(output, "\n  {e:?}");
-            output
-          })
-        );
-      }
-      "Error"
-    })
```
Comment (bartlomieju) [inline cli/errors.rs:127]: Why was this removed?

Comment (dsherret) [reply cli/errors.rs:127]: I suggested to. It doesn't seem useful?

Comment (dsherret) [reply cli/errors.rs:127]: See https://github.com/denoland/deno/pull/24721

Comment (bartlomieju) [reply cli/errors.rs:127]: Okay, that makes sense
